src/tests/test_typeconversions.py

```diff
@@ -327,3 +327,20 @@ def test_javascript_error_back_to_js(selenium):
         return pyodide.globals["py_err"] === err
         """
     )
+
+
+def test_memoryview_conversion(selenium):
+    selenium.run(
+        """
+        import array
+        a = array.array("Q", [1,2,3])
+        """
+    )
+    selenium.run_js(
+        """
+        pyodide.globals.a
+        if(pyodide._module._PyErr_Occurred()){
```

Comment:
This check would be made redundant by #1113, right?

Reply:
If so, I think the correct course of action is to merge this and then remove this line in #1113

Reply:
Either we should remove this here and keep the line in #1113 or keep the line here and close #1113.

Reply:
I meant merge this test, and then in #1113 remove this particular line in this test

Reply:
Oh yeah. I agree.